extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -222,6 +237,39 @@ const stateComparisonFunctions = {
     (old, curr) => old.element !== curr.element || old.state !== curr.state,
   [StateProperty.NAVIGATION_PATH]: (old, curr) => old.length !== curr.length,
   [StateProperty.PAGE_IDS]: (old, curr) => old.length !== curr.length,
+  [StateProperty.INTERACTIVE_RESULTS_STATE]: (old, curr) => {
+    return old.finished != curr.finished;
+  },
```

Comment:
Now we just need to update if it's finished, but we can dispatch events for other things in the future